docs/feature-flagging/randomization-sdk/server-sdks/python.md

```diff
@@ -0,0 +1,80 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Python
+
+### 1. Install the SDK
+
+Install the SDK with PIP:
+
+```bash
+pip install eppo-server-sdk
+```
+
+### 2. Define an Assignment Logger
+
+The SDK requires an assignment logger to be passed on initialization. The SDK invokes the logger to capture assignment data whenever a variation is assigned. The below code example shows how to integrate the SDK with [Segment](https://segment.com/docs/) for logging events. You could also use your own logging system; the only requirement is that the SDK receives a `log_assignment` function.
+
+Define an implementation of the Eppo `AssignmentLogger` interface. This interface has one function: `log_assignment`.
+
+```python
+from eppo_client.assignment_logger import AssignmentLogger
+import analytics
+
+# Connect to Segment (or your own event-tracking system) 
+analytics.write_key = "<SEGMENT_WRITE_KEY>"
+
+class SegmentAssignmentLogger(AssignmentLogger):
+		def log_assignment(self, assignment):
+			analytics.track(assignment["subject"], "Eppo Randomization Assignment", assignment)
+```
+
+The SDK will invoke the `log_assignment` function with an `assignment` object that contains the following fields:
+
+| Field | Description | Example |
+| --------- | ------- | ---------- |
+| `experiment` (string) | An Eppo experiment key | "recommendation_algo" |
+| `subject` (string) | An identifier of the subject or user assigned to the experiment variation | UUID |
+| `variation` (string) | The experiment variation the subject was assigned to | "control" |
+| `timestamp` (string) | The time when the subject was assigned to the variation | 2021-06-22T17:35:12.000Z |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }` |
```

Comment:
This one was not implemented as snake case in the code. The reason is so that the event tracker receives a consistent schema, as it might be receiving events from the frontend SDK (camelCase) as well as the python one.